lib/iarc/serializers.py

```diff
@@ -0,0 +1,66 @@
+from rest_framework.exceptions import ParseError
+
+from mkt.constants.iarc_mappings import (BODIES, DESCS_V2, INTERACTIVES_V2,
+                                         RATINGS)
+from mkt.site.utils import cached_property
+
+
+class IARCV2RatingListSerializer(object):
+    """Class that is used to de-serialize json sent/received from IARC to our
```

Comment:
It should. I tried it with the `SearchCerts` demo, and from the documentation it looks like the same format is used for `GetRatingChanges` and `PushCert` as well (we will still need to generate the response for `PushCert` with something else)